packages/sync-server/src/__test__/example.test.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2025 Beyond Essential Systems Pty Ltd
+ *
+ */
+
```

Comment:
No need for these anymore; we’ve stopped including them 🪄💣 

Did these also come from a template? If so, I can fix that up at the same time I update the manifest file template

```suggestion
/*
 * Tupaia
 * Copyright (c) 2017 - 2025 Beyond Essential Systems Pty Ltd
 *
 */

```

Reply:
Oh not really, I was manually copying this. I won't add it anymore